MainMenu/src/mainmenu/model/MainMenuModel.java

```diff
@@ -0,0 +1,31 @@
+package mainmenu.model;
+
+import ctc.model.CentralTrafficControl;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+public class MainMenuModel {
```

Comment:
Shouldn't this just be MainMenu/MainMenu.java following our convention of models?